README.markdown

```diff
@@ -38,9 +44,9 @@ bootstrapped:
 
     git clone git://github.com/rspec/rspec-dev.git
     cd rspec-dev
-    bundle install --binstubs
-    bin/rake setup
-    bin/rake # runs tests in every repository
+    bundle install --path=vendor/bundle
```

Comment:
That said maybe I should either add an explanation about `--path=vendor/bundle` or leave it out.  And running with `--binstubs` on Windows doesn't fail, it's just that the stubs themselves won't work.